### ISSUE TITLE
Missing chameleon, squirrel, and film datasets

### ISSUE THREAD

**Issue by Xiuyu-Li** [jianhao2016/GPRGNN#1]
Hi,
It seems like currently there are no processed chameleon, squirrel, and film datasets in the repo, and related experiments could not be directly run.

Also, what are the differences between the `dataset_heterophily` and `WebKB` class? I managed to run those datasets by directly modifying the `WebKB` class using the downloaded raw files from the Geom-GCN repo (film dataset is processed acccording to their code):
```python
data = f.read().split('\n')[1:-1]
if self.name == 'film':
    x = []
    for r in data:
        feat_one_hot = np.zeros(932)
        feat_one_hot[[int(v) for v in r.split('\t')[1].split(',')]] = 1
        x.append(feat_one_hot)
else:
    x = [[float(v) for v in r.split('\t')[1].split(',')] for r in data]
x = torch.tensor(x, dtype=torch.float)
```
Does the `dataset_heterophily` class process the raw files differently from what I have done? Thank you!

**Comment by elichienxD** [comment]
Hi Xiuyu,

For chameleon, squirrel, and film datasets we directly downloaded from the Geom-GCN repo. We use dataset_heterophily() only for normalizing node features with option T.NormalizeFeatures(). This is done for all datasets including Cora, Citeseer...etc. 

To run on those datasets, first, you need to download the raw data from the Geom-GCN repo (geom-gcn/new_data/). After saving them into Pytorch geometric data format, you can use dataset_heterophily() to access them.

For Cornell and Texas datasets, we directly use WebKB() to access them, which is provided by Pytorch geometric library itself. 

Hope this explanation help!

Eli

**Comment by jianhao2016** [comment]
Hi Xiuyu, 

I have added the datasets (squirrel, film, and chameleon) to the folder `data/`.  You should be able to directly access them now.

**Comment by Xiuyu-Li** [comment]
Hi Eli,

Thanks for the explanation and datasets!
